### PR TITLE
Add Tabbed layout

### PR DESCRIPTION
**Pull request by rohithbalaji123**
Part of #612 

### Description
Add a generic tabbed layout component.

### Screenshots
image

### ToDo:
- [x] Add tests

### CODE REVIEW

File: frontend/src/component/layouts/TabbedLayout.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+@import '../styles/font';
+
+.tab-layout {
+  display: flex;
+  height: 100%;
+  width: 100%;
+
+  .tab-headers {
+    flex: 0 0;
+  }
```
Comment (byliuyang) [inline frontend/src/component/layouts/TabbedLayout.scss:10]: I think we should remove this and leave tab content creators to decide whatever they want to do.

File: frontend/src/component/layouts/TabbedLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import React, { Component, ReactChild } from 'react';
```
Comment (byliuyang) [inline frontend/src/component/layouts/TabbedLayout.tsx:1]: I think tab layout is more like this: https://material.angular.io/components/tabs/overview
This component has drawer. I am not sure whether it qualifies for tab component. I recommend directly put them inside admin page.

What are your thoughts on this?

Comment (byliuyang) [reply frontend/src/component/layouts/TabbedLayout.tsx:1]: To clarify, I recommend closing this PR and create an empty admin page PR instead.

Comment (rohithbalaji123) [reply frontend/src/component/layouts/TabbedLayout.tsx:1]: Okay, got it!

**Pull request closed without merging**